app/assets/stylesheets/shared/textile.scss

```diff
@@ -263,7 +263,7 @@
         }
 
         &.active {
-          background: $grayDark;
+          background: darken($grayDark, 10%);
```

Comment:
Rhyme or reason to some of these changes and percent values?

Reply:
This is to adjust the output color of the variable to meet the WCAG 2.0 Level AA standards.